components/apimgt/org.wso2.carbon.apimgt.hostobjects/src/main/java/org/wso2/carbon/apimgt/hostobjects/APIProviderHostObject.java

```diff
@@ -297,11 +296,11 @@ public static String jsFunction_getHTTPsURL(Context cx, Scriptable thisObj,
      * @param thisObj Scriptable object
      * @param args    Passing arguments
      * @param funObj  Function object
-     * @return true if the API was added successfully
+     * @return json String of environments failed
      * @throws APIManagementException Wrapped exception by org.wso2.carbon.apimgt.api.APIManagementException
      */
-    public static boolean jsFunction_manageAPI(Context cx, Scriptable thisObj,
-			Object[] args,	Function funObj) throws APIManagementException, ScriptException {
+    public static String jsFunction_manageAPI(Context cx, Scriptable thisObj,

```

Comment:
Hi kasun,

This is the way of method naming we used in hostobject where jaggery used.
Thanks
tharindu